WcaOnRails/app/controllers/translations_controller.rb

```diff
@@ -42,6 +42,14 @@ def update
     message = "Update #{locale} translation."
     content_digest = Digest::SHA1.hexdigest(content)
     branch_name = "translation-#{locale}-#{content_digest}"
+    # We create a branch pointing to upstream master SHA. This stopped working
+    # unless the commit exists in the fork repository, so we always sync first.
+    # Octokit doesn't expost a method for this, so we make a regular request.
+    RestClient.post("https://api.github.com/repos/#{origin_repo}/merge-upstream", '{"branch":"master"}', {
+                      accept: "application/vnd.github+json",
+                      authorization: "Bearer #{AppSecrets.GITHUB_CREATE_PR_ACCESS_TOKEN}",
+                      x_github_api_version: "2022-11-28",
+                    })
```

Comment:
```suggestion
    # We create a branch pointing to upstream master SHA. This stopped working
    # unless the commit exists in the fork repository, so we always sync first.
    # Octokit doesn't expost a direct method for this, so we monkey-patch our request.
    Octokit.client.post "#{Octokit::Repository.path upstream_repo}/merge-upstream", { branch: 'master' }
```

Reply:
(FYI this `Octokit.client.post` approach with a string-templated URL based on `Octokit::Repository.path` is exactly how Octokit internally routes pretty much all API requests that it exposes convenience methods for)